ops/terraform/modules/resources/jenkins/main.tf

```diff
@@ -1,4 +1,6 @@
 locals {
+  azs         = ["us-east-1a"]
+  env_config  = {env=var.env_config.env, tags=var.env_config.tags, azs=local.azs}
```

Comment:
This is a bit confusing having two AZ variables.  It would be good to standardize how we do this.